content/en/getting_started/session_replay/_index.md

```diff
@@ -0,0 +1,140 @@
+      tag: 'Blog'
+      text: 'Use Datadog Session Replay to view real-time user journeys'
+    - link: '/real_user_monitoring/browser/troubleshooting/'
+      tag: 'Documentation'
+      text: 'Troubleshooting'
+---
+
+{{< img src="/getting_started/session_replay/preview.mp4" alt="Preview of Session Replay" style="width:100%" video=true >}}
+
+## Overview
+
+**Session Replay** is a visual tool that recreates user sessions from your applications, giving you an in-depth, video-like view of how customers are actually interacting with your product. Session replay enriches traditional, quantitative data—such as click counts, bounce rates, and page view metrics—with the qualitative context you need to analyze your users' actions.
+
+This page demonstrates how to get started with Session Replay in Datadog. If you haven't already, [create a Datadog account][1].
+
+## Configure Session Replays
+
+Session Replay is available in the RUM Browser SDK. To start collecting data for Session Replay, ensure that you have done the following:
+
+1. Set up [Datadog RUM Browser Monitoring][7] by creating a RUM application (ensure you toggle **Session Replay Enabled** to access replay recordings).
+2. Generate a **Client Token**. 
+3. Initialize the RUM Browser SDK.
+
+Until Datadog starts receiving data, your application appears as `pending` on the **RUM Applications** page.
+
+For more detailed instructions on collecting Session Replay data, follow the [RUM Getting Started guide][2] for your application. For setup in mobile environments, see [Mobile Session Replay][3].
+
+## Find particular Session Replays
+
+Once you're collecting Session Replay data, navigate to the [RUM Explorer][4] and select **Session Replay available** to see all sessions with a replay attached to it. You can visualize this data as a **List**, **Timeseries**, **Top List**, **Table**, **Distribution**, **Geomap**, **Funnel**, **Tree Map**, or **Pie Chart**. 
+
+{{< img src="/getting_started/session_replay/session-replay-available.png" alt="Session Replay available button, as well as visualization options" style="width:100%" >}}
+
+You can filter Sessions using **facets**. Filtering by [facet][5] is helpful if you are searching for specific information, such as a particular user or device type. 
+
+{{< img src="/getting_started/session_replay/facets-views.png" alt="Filtering by facet" style="width:100%" >}}
+
+You are also able to filter by [view][6], as well as customize and pin views. This can be helpful in the case that you know where the issue lies.
+
+{{< img src="/getting_started/session_replay/pinned-views.png" alt="Views dropdown" style="width:100%" >}}
+
+## Examine a user journey
+
+The **User Journey** functions as an event timeline, and is tracked on the right side of the page. Starting from the beginning of a replay, you can watch everything that a particular user did. You can navigate to any moment in the user journey by clicking on an event.
```

Comment:
you haven't mentioned here how to start or move around the SR playback, which you should. They can either click points in the user journey list, or they can click the play button on the SR "video"